examples/js/lights/LightProbeGenerator.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * @author WestLangley / http://github.com/WestLangley
+ */
+
+THREE.LightProbeGenerator = {
+
+	fromAmbientLight: function ( light ) {
```

Comment:
I am not a fan of this API, to be honest. What is the workflow where one would instantiate an ambient light (or hemisphere light) and then use it to instantiate a light probe?

Reply:
I think it's just temporal. Something like `new THREE.AmbientLightProbe()` sounds better to me.

Reply:
That was actually the direction I was heading...

I do not have a problem with moving `.fromCubeTexture()` to the examples, but I prefer #16298.

I think we have time to fix this before the next release.